numba/cuda/cudadrv/nvvm.py

```diff
@@ -319,6 +323,8 @@ def find_closest_arch(mycc):
     :return: Closest supported CC as a tuple ``(MAJOR, MINOR)``
     """
     supported_cc = get_supported_ccs()
+    if not supported_cc:
+        raise NvvmSupportError("No supported CUDA Toolkit found")
```

Comment:
This is a duplicate of the condition 2 lines down?

Reply:
Yeah, it got messed up in a merge - I did something very similar here to #6227 here. Will fix.